models/database.go

```diff
@@ -184,6 +184,12 @@ var databaseSchema = [][]string{
 		// e'\n' to treat \n as a newline, not as two characters
 		`UPDATE nodes SET machine_id = trim(e'\n' from machine_id) WHERE machine_id IS NOT NULL`,
 	},
+
+	6: {
+		`ALTER TABLE agents
```

Comment:
We can't keep columns with default values in DB. We should remove defaults; see migration 4 as an example.